sdk/appconfiguration/app-configuration/swagger/swagger.md

```diff
@@ -7,92 +7,92 @@ package-name: app-configuration
 title: AppConfiguration
 description: App Configuration client
 enable-xml: true
-add-credentials: true
+add-credentials: false
 generate-metadata: false
 license-header: MICROSOFT_MIT_NO_VERSION
 output-folder: ../src/generated
 input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/appconfiguration/data-plane/Microsoft.AppConfiguration/stable/1.0/appconfiguration.json
 model-date-time-as-string: false
 optional-response-headers: true
 sample-generation: false
+core-http-compat-mode: true
 use-extension:
-  "@autorest/typescript": "6.0.0-dev.20210111.1"
+  "@autorest/typescript": "6.0.0-beta.16"
 disable-async-iterators: true
 api-version-parameter: choice
 v3: true
 hide-clients: true
```

Comment:
Add package-version option to generate client properly.

Reply:
Need to update the metadata section in package.json too to help the bot auto-increment versions